src/NUnit.TestAdapter.Tests.Acceptance/AcceptanceTests.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
```

Comment:
🤦‍♂️ Thanks for thinking of that.

Reply:
Hmm, what happens when I move the code between repos? The copyright messages will all change.

@OsirisTerje, do you have a preference for or against copyright headers in this repo?

Reply:
We have the standard of having that copyright notice, something Charlie introduced and afaik it should be there for legal reasons.  Some files may be missing it, I am fixing them when I find them.

Reply:
We have a standard, I think, that we will __have__ copyright notices, that they will reflect the individual copyright holder and will include the license in use.

Since this is an entirely new thing that will be used in various projects,, it could have it's own standard format. Possibly, it could be in its own repo - or at least the underlying classes used to implement the acceptance tests could.

As a reminder, I was once the sole copyright holder for all NUnit 3 code. When Rob's level of code contribution to the project began to approach my own in quantity and importance, I added him as a copyright holder. Similarly for Terje and the adapters. That was a grant, which it was in my power to make. I did it because it made sense to me but there was no requirement I should do so.

Although IANAL, it seems clear that Joseph holds copyright to his work at the time he creates it. That's certainly true under US law. He can keep it or he can donate the code separately to each project where it then would fall under the copyright of that project. Normally, this question doesn't come up because contributions are incremental increases in project functionality. In this case, however, we have an entire new tool, which is used by the project rather than being distributed to users as part of the adapter, so it seems like it can be handled in a variety of ways.

I have no opinion as to how it should be handled. In fact, I do not believe there is any "should" here. It's an arbitrary decision.

Reply:
Wait a minute, I have the opportunity to decide what the license will be? I am so tempted to write a license that requires that no source file under that license may contain a license header! 😜 

I don't know what the significance is of me submitting the code and already having it merged, but my intention is to reuse this code in other NUnit repos, and very likely outside the NUnit organization. As unrealistic as I know this wish is, quite honestly I wish I didn't have to spend brain cells on any of this.

No matter whose name is on the copyright, as long as the stated license is MIT, anyone will be able to copy and modify this code anywhere they like so long as they keep the copyright notice and license intact. That's pretty much all I care about. Though if I'm allowed a preference on this, one-liners > half-screen license headers, and no headers > one-liners.

**Option A**: The copyright notice will stay as "Charlie Poole, Terje Sandstrom," even when moved to other NUnit or non-NUnit repositories, unless they decide to give the copyright to someone else. (A third name, for example.)

**Option B**: The copyright will stay as "Joseph Musser," even when moved to other NUnit or non-NUnit repositories, unless I decide to give the copyright to someone else.

@OsirisTerje I'm solidly 50/50 on this one, so please tell me what you want me to do.

Reply:
@jnm2 I see you're at least 50% tongue-in-cheek here but I'm enough of an open source licensing geek that I want to take it a beat further. :smile: I'll continue offline.